src/main/java/step3/domain/Cars.java

```diff
@@ -0,0 +1,28 @@
+package step3.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Cars {
+  private final List<Car> carList;
```

Comment:
```suggestion
  private final List<Car> cars;
```
타입이 있으니 변수명에 List는 제외해도 괜찮겠네요